msal/src/Microsoft.Identity.Client/Exceptions/MsalExceptionService.cs

```diff
@@ -36,7 +36,7 @@ namespace Microsoft.Identity.Client
     /// <summary>
     /// Implementation of the <see cref="CoreExceptionService"/> that throws <see cref="MsalException"/>
     /// </summary>
-    internal class MsalExceptionFactory : CoreExceptionService
+    internal class MsalExceptionService : CoreExceptionService
```

Comment:
>MsalExceptionService [](start = 19, length = 20)

this naming is confusing... sorry, perhaps going back to factory is better...

Reply:
I'll revert to the old name (*ExceptionFactory) and get this checked in